operate/schema/src/test/java/io/camunda/operate/enties/ListenerStateTest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.operate.enties;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import io.camunda.operate.entities.ListenerState;
+import org.junit.jupiter.api.Test;
+
+public class ListenerStateTest {
+
+  @Test
+  public void convertStatesFromZeebeJobs() {
```

Comment:
What about checking for
```java
case "COMPLETED" -> COMPLETED;
case "TIMED_OUT" -> TIMED_OUT;
case "CANCELED" -> CANCELED;
```

Reply:
I originally left those out because they get implicitly tested in the integration tests, but will add them (also to have it there in the spirit of "tests can be a form of documentation").